types/restify-cookies/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for restify-cookies 0.2
+// Project: https://github.com/nathschmidt/restify-cookies
+// Definitions by: weekens <https://github.com/weekens>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as restify from 'restify';
+
+declare module 'restify' {
+  interface CookieOptions {
+    encode?: (input: string) => string; // tslint:disable-line:prefer-method-signature
+    maxAge?: number;
+    domain?: string;
+    path?: string;
+    expires?: string;
```

Comment:
@weekens it should be Date can you fix it?

restify-cookies i using cookie.serialize
https://github.com/nathschmidt/restify-cookies/blob/v0.2.3/index.js#L50

You could use existing cookie definitions from here:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cookie/index.d.ts#L37

Reply:
@piorek Sure, I'll fix that.

I looked at the definitions you provided. In fact, not all fields from that definition are used in restify-cookies (for example, `sameSite` is not used).

Reply:
Created new PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/20752

Reply:
@weekens Thx